demos/2022.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: demos
+title: Community Demos 2022
+---
+
+# Community Demos (2022)
+------------------------------
+
+Every few weeks we host a Community Demo to showcase new & interesting developments from the Foreman
+community.
+We encourage participation from any member of the community (although you do need a
+Google account), so if you’ve been working on something cool, please do come show it off.
+If you'd like to watch the community demo, they're usually on a Thursday and we always list them in
+advance on our [Community Events Calendar](https://community.theforeman.org/c/events/l/calendar).
+
+Here is a list of every demo that has taken place this year.
+Our demos are split into two sections: a user-focused section and a developer-focused section.
+
+If you've any questions or feedback, we welcome any feedback in our
+[community](https://community.theforeman.org)
+
+You can find older demos here:
+* [Foreman Community Demos](/demos)
+
+## User-focused Demos - Foreman
+### [Demo 105](https://youtu.be/uk9PvytPhsA)
+* [Modularized Documentation](https://youtu.be/uk9PvytPhsA?t=843)
+* [Template helpers `truthy?` and `falsy?`](https://youtu.be/uk9PvytPhsA?t=542)
+* [FDI EL8 build 4.0](https://youtu.be/uk9PvytPhsA?t=1591)
+* [New report format update](https://youtu.be/uk9PvytPhsA?t=1850)
+* [New reports UX design](https://youtu.be/uk9PvytPhsA?t=2149)
+
+### [Demo 106](https://youtu.be/It4vn5_stCQ)
+* [Registration and host_init_config templates previews](https://youtu.be/It4vn5_stCQ?t=358)
+* [Officially support only the recent versions of Firefox, Safari, Edge, Chrome/Chromium browsers](https://youtu.be/It4vn5_stCQ?t=726)
+* [Puppet tab demo](https://youtu.be/It4vn5_stCQ?t=912)
+
+### [Demo 107](https://youtu.be/mFfz6p_7P4Y)
+* [New Reports plugin & Reports tab on host page](https://youtu.be/mFfz6p_7P4Y?t=186)
+* [new reports dashboard widgets](https://youtu.be/mFfz6p_7P4Y?t=372)
+
+### [Demo 108](https://youtu.be/ykQFM7aFczY)
+* [2 Bug fixes in the bookmarks](https://youtu.be/ykQFM7aFczY?t=1670)
+* [Import ansible playbooks](https://youtu.be/ykQFM7aFczY?t=1788)
+* [New features in host page](https://youtu.be/ykQFM7aFczY?t=1987)
+* [DSL autocompletion in templates](https://youtu.be/ykQFM7aFczY?t=2311)
+
+## User-focused Demos - Foreman & Katello plugin
```

Comment:
It took me a second to understand the user focused demos were split in vanilla Foreman and Katello installed demos. I wonder if this would clarify it a bit.

```suggestion
## User-focused Demos - Foreman with Katello installed
```